hs_core/hydroshare/hs_bagit.py

```diff
@@ -222,6 +222,8 @@ def create_bag(resource):
 
     istorage.setAVU(resource.short_id, "resourceType", resource._meta.object_name)
 
+    # delete if there exists any bags for the resource
+    resource.bags.all().delete()
```

Comment:
@pkdash I am wondering whether doing the following is better considering there might not be existing bags in which case resource.bags.all() might be None:
```
for bag in resource.bags.all():
    bag.delete()
```